src/libData/BlockData/BlockHeader/TxBlockHeader.cpp

```diff
@@ -23,7 +23,6 @@ using namespace boost::multiprecision;
 TxBlockHeader::TxBlockHeader()
 {
     m_blockNum = (boost::multiprecision::uint256_t)-1;
```

Comment:
Coding style of constructor initialization should be consistent.
At least, we should choose one style and follow it inside same file.
For example, replace this line to
TxBlockHeader::TxBlockHeader()
: m_blockNum((boost::multiprecision::uint256_t)-1)
{}